assets/dev/js/editor/regions/navigator/navigator.js

```diff
@@ -91,15 +91,12 @@ export default class extends BaseRegion {
 		this.show( new NavigatorLayout() );
 
 		this.$el.draggable( this.getDraggableOptions() );
-
 		this.$el.resizable( this.getResizableOptions() );
 	}
 
 	open( model ) {
 		// If open once.
 		if ( ! this.opened ) {
-			this.initLayout();
-
 			this.opened = true;
 		}
```

Comment:
These lines are now redundant

Reply:
Why did you resolve this comment?

Reply:
redundant means they not needed anymore?

Reply:
@kobizz please explain what what you mean or what you want me to do

Reply:
The whole block of code should be removed:
```
// If open once.
if ( ! this.opened ) {
	this.opened = true;
}